Lib/swig.swg

```diff
@@ -678,7 +678,7 @@ template<typename T> class SwigValueWrapper {
 public:
   SwigValueWrapper() : pointer(0) { }
   SwigValueWrapper& operator=(const T& t) { SwigSmartPointer tmp(new T(t)); pointer = tmp; return *this; }
-#if __cplusplus >=201103L
+#if defined(__cplusplus) && __cplusplus >=201103L
```

Comment:
This isn't needed - it's already checked at line 665 above, and this is a class definition so clearly C++-only code.